content/influxdb/cloud-dedicated/reference/cli/influxctl/database/create.md

```diff
@@ -102,3 +114,20 @@ influxctl database create \
   --max-columns 150 \
   mydb
 ```
+
+### Create a database with with a custom partition template
+
+The following example creates a new `mydb` database and applies a partition
+template that partitions by two tags (`room` and `sensor-type`) and by week using
+the time format `%Y wk:%W`:
```

Comment:
In define-custom-partitions you show `wk:%U`, and `wk:%W`, I assume either work as my understanding is the difference is when the day of the week starts, but do we want to be consistent?

Same question below

Reply:
Nice catch. I'll update.